backend/lib/azimutt_web/admin/event/_events_table.html.heex

```diff
@@ -1,18 +1,18 @@
-<div class="flex flex-col border-l border-1 border-slate-800 pl-3">
-    <div class="py-2 flex justify-between">
+<div class="flex flex-col pl-3 border-l border-1 border-slate-800">
```

Comment:
is it done by a tool or manually?

Reply:
faudrait l'installer dans le projet pour que tout le monde l'ai du coup

Reply:
https://tailwindcss.com/blog/automatic-class-sorting-with-prettier